luigi_monitor/luigi_monitor.py

```diff
@@ -70,6 +75,9 @@ def _event_task_message(self, event):
             if event == MISSING:
                 event_tasks.append(task)
 
+            if event == PRESENT:
```

Comment:
This is the same as the if block above for MISSING.

You can replace with:
```
if event in [MISSING, PRESENT]:
    event_tasks.append(task)
```

Reply:
👍